src/Schema/Blueprint.php

```diff
@@ -51,47 +52,44 @@ public function dropPrimary($index = null)
 
     /**
      * @inheritDoc
-     * @return never
      */
     public function increments($column)
     {
-        $this->markAsNotSupported('AUTO_INCREMENT');
+        return $this->uuid($column)
+            ->generateUuid()
+            ->primary();
```

Comment:
README.md doesn't mention that increments column will be primary.

It would be better to mention it in the README or not to make it primary.

Reply:
It shouldn't have to. That's the expected behavior.

https://github.com/laravel/framework/blob/10.x/src/Illuminate/Database/Schema/Grammars/MySqlGrammar.php#L1308-L1313

Reply:
Also stated in the docs.

https://laravel.com/docs/10.x/migrations#column-method-increments

> The increments method creates an auto-incrementing UNSIGNED INTEGER equivalent column as a primary key:

Reply:
Thank you for reference.

I had read [Schema/Blueprint](https://github.com/laravel/framework/blob/10.x/src/Illuminate/Database/Schema/Blueprint.php#L706) and didn't realize it.

Reply:
I see what you mean though, description for it being set to primary key has been added to the docs.